source/documentation/at-rules/mixin.liquid

```diff
@@ -62,25 +63,30 @@ nav ul {
       left: -2px
       right: 2em
 
+
+
+
```

Comment:
@sanajaved7 I think these extra lines in Sass code examples are often intentional, to line up similar lines between SCSS and Sass examples.